core/base-service/auth-helper.js

```diff
@@ -74,7 +74,7 @@ class AuthHelper {
   }
 
   static _isInsecureSslRequest({ options = {} }) {
-    const { strictSSL = true } = options
+    const strictSSL = options?.https?.rejectUnauthorized ?? true
```

Comment:
TIL Node supports an optional chaining operator

Reply:
Yeah it was introduced in Node 14. This might be the first time we've used it.